client/app/components/packages/landuse-form/show.hbs

```diff
@@ -605,16 +605,15 @@
               </Ui::Answer>
             {{/if}}
 
-            <Ui::Answer @beside={{true}} as |A|>
-              <A.Prompt>
-                This is a follow-up Action. The previous approval Action Code:
-              </A.Prompt>
-              <A.Field>
-                {{optionset 'landuseAction' 'actions' 'label' landuseAction.dcpPreviouslyapprovedactioncode}}
-              </A.Field>
-            </Ui::Answer>
-
-            {{#if this.projectHasRequiredActionsAndFollowUpYes}}
```

Comment:
Perhaps the issue is that this getter `projectHasRequiredActionsAndFollowUpYes` needs to be added to landuse-form/show.js, just like how it is in landuse-form/edit.js?

Reply:
That getter is unnecessary.  If landuseAction.dcpPreviouslyapprovedactioncode is not null, we want to show it and all of the responses that had previously been in the {{#if this.projectHasRequiredActionsAndFollowUpYes}} block.  If landuseAction.dcpPreviouslyapprovedactioncode is null, then we don't want to show them.

Reply:
Hmm okay I was concerned because this getting takes into account `dcpFollowuptopreviousaction` , however looking at the action it looks like the value of that field just reflects whether dcpPreviouslyapprovedactioncode is filled.